worldedit-core/src/main/resources/lang/strings.json

```
@@ -29,7 +29,7 @@
     "worldedit.brush.operation.equip": "Set brush to {0}.",
     "worldedit.brush.none.equip": "Brush unbound from your current item.",
 
-    "worldedit.setbiome.changed": "Biomes were changed in {0} columns. You may have to rejoin your game (or close and reopen your world) to see a change.",
+    "worldedit.setbiome.changed": "Biomes were changed for {0} blocks. You may have to rejoin your game (or close and reopen your world) to see a change.",
```

Might be better to say:
```suggestion
    "worldedit.setbiome.changed": "Biomes were changed for approximately {0} blocks. You may have to rejoin your game (or close and reopen your world) to see a change.",
```